Okta-Group-Application-Association/src/handlers.ts

```diff
@@ -40,12 +36,11 @@ class Resource extends AbstractOktaResource<ResourceModel, GroupApplicationAssoc
     }
 
     async update(model: ResourceModel, typeConfiguration: TypeConfigurationModel): Promise<ResourceModel> {
-        // Nothing is updatable
-        return model;
+        throw new NotUpdatable();
```

Comment:
Does this require any of the test to be excluded?

Reply:
Yes, I ran with the following:

```
 "TestDetails": [
"TestRead, test_read_return_all_schema_properties_using_one_or_more_inputs",
"TestRead, test_governance",
"TestRead, test_read_invokable",
"TestRead, test_read_return_write_only_properties",
"TestRead, test_read_return_readonly_properties",
"TestRead, test_read_input_output_negative_match",
"TestCreate, test_create_create",
"TestDelete, test_delete_read",
"TestDelete, test_delete_delete"
]
```